docs/developing.rst

```diff
@@ -0,0 +1,58 @@
+Developing
+==========
+
+Install the requirements
+
+::
+
+    $ pip install -r requirements.txt
```

Comment:
There was up until a commit or two ago. Just changed the name to requirements-dev.txt and forgot to change it here.